test/testautomation_video.c

```diff
@@ -1762,6 +1829,7 @@ static int video_setWindowCenteredOnDisplay(void *arg)
 
                 /* Leave fullscreen desktop */
                 result = SDL_SetWindowFullscreen(window, SDL_FALSE);
+                SDL_SyncWindow(window);
                 SDLTest_AssertCheck(result == 0, "Verify return value; expected: 0, got: %d", result);
```

Comment:
Should the sync be before or after the assertcheck? Either way it should probably be consistent with the change on line 1809, or have a comment explaining why it matters.

Reply:
After, and there were a few other places where the result of the sync wasn't being checked properly either. Fixed.